easybuild/easyconfigs/s/SCALCE/SCALCE-2.7-GCC-4.8.2.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'MakeCp'
+
+name = 'SCALCE'
+version = '2.7'
+
+homepage = 'http://scalce.sourceforge.net/Home'
+description = """SCALCE [skeɪlz] is a FASTQ compression tool that uses locally consistet parsing
+ to obtain better compression rate. SCALCE has been specifically designed for Illumina reads but it can
+ handle other technologies (that generate base pair reads) if the read length is the same throughout the file."""
+
+toolchain = {'name': 'GCC', 'version': '4.8.2'}
+
+# eg. http://sourceforge.net/projects/scalce/files/2.7/scalce-v2.7-src.tar.gz/download
+sources = ['%(namelower)s-v%(version)s-src.tar.gz']
+source_urls = [SOURCEFORGE_SOURCE]
+
+preconfigopts = 'cd zlib-1.2.8 &&'
```

Comment:
@boegel: dunno; 
SCALCE carries with it its own version of zlib so I made the assumption there must be a cause;
however, it's only an assumption in the name of safety...